content/en/docs/usage/kraft_advanced.md

```diff
@@ -0,0 +1,732 @@
+#
+make: Leaving directory '/home/stefan/.unikraft/unikraft'
+```
+
+### Overview of the `kraft configure` Command
+
+You can get a full list of features by running `kraft configure -h` or `kraft configure --help`.
+
+```text
+Usage: kraft configure [OPTIONS]
+
+  Configure the unikernel using the KConfig options set in the kraft.yaml
+  file.  Alternatively, you can use the -k|--menuconfig flag to open the TUI
+  to manually select the configuration for this unikernel.
+
+  When the unikernel is configured, a .config file is written to the working
+  directory with the selected KConfig options.
+
+Options:
+  -p, --plat PLAT         Target platform.
+  -t, --target TARGET     Target name.
+  -m, --arch ARCH         Target architecture.
+  -F, --force             Force writing new configuration.
+  -k, --menuconfig        Use Unikraft's ncurses Kconfig editor.
+  -w, --workdir PATH      Specify an alternative directory for the application
+			  [default is cwd].
+  -y, --yes KOPTION       Specify an option to enable.
+  -n, --no KOPTION        Specify an option to disable.
+  -s, --set KOPTION       Set an option's value.
+  -u, --use-version COMP  Use the specified version for the component, e.g.
+			  -u unikraft@staging (will override kraft.yaml).
+  -h, --help              Show this message and exit.
+```
+
+## `kraft build`
+
+You can build your application by using `kraft build`.
+You can use the `--fetch / --no-fetch` and `--prepare / --no-prepare` flags to run the fetch and prepare steps before building.
+You can use the `-j, --fast` option.
+This will pass the `-j` option to `make`, resulting in running multiple jobs that can run simultaneously.
+If you want to force the overwrite of the existing build, use `-F, --force`.
+
+```console
+$ kraft build --no-fetch --no-prepare -F -j
+CC      libukdebug: hexdump.o
+LD      libukdebug.ld.o
+OBJCOPY libukdebug.o
+[...]
+Successfully built unikernels:
+
+  => build/hello_kvm-x86_64
+  => build/hello_kvm-x86_64.dbg (with symbols)
+
+To instantiate, use: kraft run
+```
+
+### Overview of `kraft build` command
+
+You can get a full list of features by running `kraft build -h` or `kraft build --help`:
+
+```text
+Usage: kraft build [OPTIONS] [TARGET]
+
+  Builds the Unikraft application for the target architecture and platform.
+
+Options:
+  -v, --verbose               Verbose build
+  --fetch / --no-fetch        Run fetch step before build.
+  --prepare / --no-prepare    Run prepare step before build.
+  --progress / --no-progress  Show progress of build.
+  -j, --fast                  Use all CPU cores to build the application.
+  -F, --force                 Force the build of the unikernel.
+  -h, --help                  Show this message and exit.
+
+```
+
+## `kraft run`
+
+You can run your application using `kraft run`.
+You can specify the target architecture and platform using `-m, --arch` and `-p, --plat`:
```

Comment:
```suggestion
In case of multiple existing builds (for different platforms and architectures), you will be prompted for the image to run, similarly to [`kraft configure`](#...):

"""
.... # sample prompt
"""

To run the specific image non-interactively, use the `-m, --arch` and `-p, --plat` options to specify the architecture and the platform:
```